src/main/java/com/enderio/base/data/recipe/MaterialRecipes.java

```diff
@@ -281,31 +254,24 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> recipeConsumer) {
         grindingBall(recipeConsumer, EIOItems.END_STEEL_BALL.get(), EIOItems.END_STEEL_INGOT.get());
     }
 
-    private void blockToIngots(Consumer<FinishedRecipe> recipeConsumer, Item ingot, Block block) {
-        ShapelessRecipeBuilder.shapeless(ingot, 9)
-            .requires(block.asItem())
-            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(block.asItem()))
-            .save(recipeConsumer);
-    }
-    private void ingotsToBlock(Consumer<FinishedRecipe> recipeConsumer, Block block, Item ingot) {
-        ShapelessRecipeBuilder.shapeless(block)
-            .requires(ingot,9)
-            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(block.asItem()))
-            .save(recipeConsumer, ingot.toString() + "_to_block");
-    }
-    private void ingotToNuggets(Consumer<FinishedRecipe> recipeConsumer, Item nugget, Item ingot) {
-        ShapelessRecipeBuilder.shapeless(nugget, 9)
-            .requires(ingot)
-            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(ingot))
-            .save(recipeConsumer);
-    }
-    private void nuggetstoIngot(Consumer<FinishedRecipe> recipeConsumer, Item ingot, Item nugget) {
-        ShapelessRecipeBuilder.shapeless(ingot)
-            .requires(nugget,9)
-            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(ingot))
-            .save(recipeConsumer, nugget.toString() + "_to_ingot");
-    }
-    
+	private void makeMaterialRecipes(Consumer<FinishedRecipe> recipeConsumer, Item ingot, Item nugget, Block block ) {
+		ShapelessRecipeBuilder.shapeless(ingot, 9)
+		.requires(block.asItem())
```

Comment:
Add Tab on all those calls on the builder

Reply:
also done

Reply:
The shaped calls aren't tabbed correctly